controller_manager/controller_manager/controller_manager_services.py

```diff
@@ -107,6 +108,18 @@ def reload_controller_libraries(node, controller_manager_name, force_kill):
     )
 
 
+def set_hardware_component_state(node, controller_manager_name, component_name, lifecyle_state):
```

Comment:
please reformat this using pre-commit.

Check this manual if you need help: https://rtw.stoglrobotics.de/master/guidelines/development.html#use-pre-commit-for-formatting-and-linting